components/post-header/post-header.module.scss

```diff
@@ -0,0 +1,37 @@
+.post-header {
+  display: flex;
+  flex-direction: column;
+  gap: 33px;
+  width: 83.3%;
+  margin: 0 auto;
+  @media screen and (max-width: 500px) {
+    gap: 20px;
+    width: 90%;
+  }
+}
+
+.post-title {
+  width: 86%;
+  margin: 0 auto 1em;
+  font-weight: 300;
+  font-size: 4.375em;
+  text-align: center;
```

Comment:
From the Figma it looks like the title is aligned left, no center.

Reply:
I forgot to apply the heading styles to `h1`. Regarding the font-family, on Figma the headings inside posts are all 'Studio Feixen Sans', should I change them to use the same font as the paragraphs?

Reply:
Actually I think Studio Feixen Sans and Studio Feixen are the same font 🤔 so just use the same font-family as the post titles.